docs/guide/hello-world.md

```diff
@@ -63,15 +63,16 @@ const Controller = {
 export default Controller;
 ```
 
-### 4. Setup the route
+### 4. Set up the route
 
-Next step is to create the main file for your section, called `index.js` within `hello-world`. Run the following command:
+The next step is to create the main file for your section, called `index.js` within `hello-world`.  
+Run the following command to create the file:
 
 ```
 touch client/my-sites/hello-world/index.js
 ```
 
-Here we'll require the `page` module, the My Sites controller, our own controller, and write our main route handler:
+Here we'll require the `page` module, the My Sites controller and our own controller, and write our main route handler:
```

Comment:
I may be wrong here but an 'and' felt better to me.
'I see two lists here... He're we'll require _some things_ and write our main route handler'